lib/cylc/task_remote_mgr.py

```diff
@@ -275,7 +275,7 @@ def remote_tidy(self):
                     LOG.warning(TaskRemoteMgmtError(
                         TaskRemoteMgmtError.MSG_TIDY,
                         (host, owner), ' '.join(quote(item) for item in cmd),
-                        proc.ret_code, out, err))
+                        proc.returncode, out, err))
```

Comment:
`proc` is a `subprocess.Popen` instance, not a `SubProcContext` or whatever the in-house Cylc thing is called. So the correct attribute is `.returncode` - https://docs.python.org/2.7/library/subprocess.html#subprocess.Popen.returncode